packages/frontend/src/pages/InferenceServer.spec.ts

```diff
@@ -0,0 +1,75 @@
+/**********************************************************************
```

Comment:
Missing an s in the filename -> `InferenceServers.spec.ts`

Reply:
Oh yes thanks